ecmascript/codegen/src/lib.rs

```diff
@@ -2308,15 +2308,10 @@ fn unescape(s: &str) -> String {
         }
     }
 
-    // let s = s.replace("\\\\", "\\");
-
     let mut result = String::with_capacity(s.len() * 6 / 5);
     let mut chars = s.chars();
 
-    println!("S: {:?}", s);
```

Comment:
Oh... Nice catch.
I completely forgot about it as tests passed and nothing was shown.